### ISSUE TITLE
Epubjs and upcoming readium new version

### ISSUE THREAD

**Issue by mbret** [futurepress/epub.js#1174]
Hello, I am currently using epubjs and like it so far but I came to follow the recent new of readium and I was wondering if there would way in the future to use readium components to simplify package or eventually combine some specs ?

From what I understand readium is not providing any final package to use right out of the box right now but rather some plugin that can be used to build a reader. This project seems interesting but It's hard to follow exactly what it is and how to glue pieces together, their website and pacages are a little bit hard to follow and understand.

On a more general note what do you guys think about the readium project ?

**Comment by OriIdan** [comment]
We had past experience with Readium 1.0 at that time they where providing a
full JS module similar to EPUB.js however it was not as complete as EPUB.js.
We have been in touch with the Readium project and even considered being
part of EDRlab who develops it.
I also think they are very complicated. I guess this is due to lack of
resources.
The people behind the project are very knowledgeable in EPUB and are
cooperative as much as I have seen.



-- 
Ori Idan CEO Helicon Books
http://www.heliconbooks.com

**Comment by AntonOfTheWoods** [comment]
The Readium group were very focused on other SDKs (iOS, Android, Desktop) until recently, though are now starting to think about web-oriented JS again. Have a look at https://github.com/d-i-t-a/R2D2BC, which again has some onboarding issues but is actually moving forward and has momentum behind it. Both Readium and R2D2BC have Slack channels where you can get support (https://readium.slack.com/ and https://r2d2bc.slack.com/), and SUPER friendly devs.

**Comment by mbret** [comment]
@AntonOfTheWoods any idea on how to join their slack channel ? When trying to login with my account it does not let me in. I think slack needs an invitation link or something ?

**Comment by AntonOfTheWoods** [comment]
@mbret , do you mean https://r2d2bc.slack.com/? If so, you might be right, lemme check.
